README.md

```diff
@@ -121,7 +123,7 @@ Now the relative position of the robot endpoint is calculated from the distance
 
 ### 6. Own inverse kinematic calculations- full arm mapping (map_arm branch)
 
-Deterministic joint angles can be achieved by specific mathematical formulas, which are not included in any package, so own calculation are needed. Because of the difference in size and the 7 degrees of freedom, these formulas are better calculated for the full arm tracking as for only the endpont following. The robot arm's joint limits are much bigger than that of the human arm, so a full mapping should be possible, with the angles at the shoulder, elbow and wrist identical. This resulted in deterministic joint angles with the pose of the arms identical with small limitations.
+Deterministic joint angles can be achieved by specific mathematical formulas, which are not included in any package, so own calculations are needed. Because of the difference in size and the 7 degrees of freedom, these formulas are better calculated for the full arm tracking as for only the endpoint following. The robot arm's joint limits are much bigger than that of the human arm, so a full mapping should be possible, with the angles at the shoulder, elbow and wrist identical. This resulted in deterministic joint angles with the pose of the arms identical with small limitations.
 
 ## Planning and inverse kinematics for endpoint following (5. approach on follow_endpoint branch)
```

Comment:
I would rather include this in the 5. section of the roadmap, it's a bit confusing this way.

Reply:
I meant the Planning and inverse... section